src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/annotation/packageannotation/Example1.java

```diff
@@ -9,7 +9,8 @@
 //non-compiled with javac: Compilable with Java7
 
 // xdoc section -- start
-@Deprecated // violation below
+@Deprecated
+// violation below, 'Package annotations must be in the package-info.java info.'
```

Comment:
Please move violation comment under package to not split package and annotation

Reply:
done